cluster/gce/manifests/kube-apiserver.manifest

```diff
@@ -34,11 +34,19 @@
       "httpGet": {
         "host": "127.0.0.1",
         "port": 8080,
-        "path": "/healthz"
+        "path": "/healthz?exclude=etcd"
```

Comment:
This is good - so we won't restart the pod just because etcd isn't up, but we won't mark it ready until it is genuinely ready.

Do we want to exclude the RBAC rules also (because it depends on etcd being / having been up)?

Reply:
Yeah, I was actually debating this one. I didn't want to jump the gun on that particular exclusion because I didn't have a good answer for what happens when RBAC bootstrapping fails for a reason unrelated to etcd. Regardless, getting started with an etcd exclusion should be sufficient to improve steady state recovery mechanisms (i.e. something happens with our persistent disk and etcd goes down temporarily, at least in this situation we do not automatically start crashlooping the api-server).

Reply:
I'm open to your thoughts on the matter though, I don't know the ins and outs of the RBAC boot sequence, so it's possible I am overly cautious.